moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -544,21 +539,22 @@ class PlanningSceneMonitor : private boost::noncopyable
 
   /// the amount of time to wait in between updates to the robot state
   // This field is protected by state_pending_mutex_
-  ros::WallDuration dt_state_update_;
+  std::chrono::duration<double> dt_state_update_;  // 1hz
```

Comment:
Maybe there should be some global guidance about replacing `ros::WallTime` and `ros::WallDuration` by `std::chrono::std::chrono::system_clock::time_point` and `std::chrono::duration`.

Reply:
I agree we should add an alias for that something like `moveit::TimePoint`